platform-hub-api/app/controllers/announcements_controller.rb

```diff
@@ -103,6 +103,19 @@ def unmark_sticky
     head :no_content
   end
 
+  # POST /announcements/:id/resend
+  def resend
+    if @announcement.mark_for_resend!
```

Comment:
Yep, it should be. (Figured admins may want to do that for some big announcements)